integration-tests/src/test/java/org/apache/druid/tests/TestNGGroup.java

```diff
@@ -84,6 +82,15 @@
    */
   public static final String HDFS_DEEP_STORAGE = "hdfs-deep-storage";
 
+  public static final String HADOOP_S3_TO_S3 = "hadoop-s3-to-s3-deep-storage";
```

Comment:
I think you also have to exclude this from Travis's other integration test group

Reply:
Ahhh actually you don't since we exclude the hadoop test package.

Reply:
Wait then your -Dgroups will not work. I believe if you run mvn cmd with -Dgroups it will use the suite defined in testng.xml. This suite excludes running hadoop package (org.apache.druid.tests.hadoop). I think one solution is you will have to remove the exclusion of the hadoop package from testng.xml and add all the new groups to the exclusion list in the Travis's other integration test group

Reply:
Updated as suggested